phpunit.xml

```diff
@@ -20,7 +20,7 @@
         </whitelist>
     </filter>
     <php>
-        <env name="BANNER_SDK_API_BASE_URI" value="http://display.aboalarm.de" />
+        <env name="BANNER_SDK_API_BASE_URI" value="http://media02.aboalarm.de" />
```

Comment:
I guess this is the live key, can't we take the info somehow from .env?
And I guess this will then do test entries and deletions on live. Which we should also avoid.

At least keep it empty for now. And we need to change the API key on live. Because this info is now public :-{